src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -177,6 +183,17 @@
                  [rowIdentity]="rowIdentity()"
                  [rowHeight]="'auto'">
 
+    <!-- Row Selection Template-->
+    <ng-template #rowSelectionTpl
+                 let-value="value"
+                 let-isSelected="isSelected"
+                 ngx-datatable-cell-template>
+      <input type="checkbox"
+             [attr.aria-label]="isSelected ? 'selected' : 'select'"
```

Comment:
I'm not familiar with the WCAG guidelines, but for consistenty, if one label states the outcome of clicking that element ("select"), the opposite should do the same ("unselect"). Alternatively, the label could describe the current status: "unselected" and "selected". 
```suggestion
             [attr.aria-label]="isSelected ? 'unselect' : 'select'"
```
```suggestion
             [attr.aria-label]="isSelected ? 'selected' : 'unselected'"
```

Reply:
@nSedrickm @epuertat I found a good article that backs up this logic:

https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html

_In the case of radio buttons, checkboxes, comboboxes, or similar controls that provide users with options, each option must have an appropriate label so that users know what they are actually selecting._

It's more of a general statement about how information should be conveyed, but it's good to see that the guidelines do have a clear stance on this topic.